packages/gibsonify_api/lib/src/models/recipe.dart

```diff
@@ -1,65 +1,41 @@
 import 'package:equatable/equatable.dart';
-import 'package:formz/formz.dart';
 import 'package:uuid/uuid.dart';
 import 'dart:convert';
 
+import 'measurement.dart';
 import 'recipe_ingredient.dart';
 import 'recipe_probe.dart';
 
-// TODO: rename this file to recipe.dart instead of recipe_item.dart
 class Recipe extends Equatable {
   Recipe({
-    this.recipeName = const RecipeName.pure(),
+    this.recipeName,
     String? recipeNumber,
     this.recipeType = "",
-    this.recipeVolume = const RecipeVolume.pure(),
+    List<Measurement>? measurements,
     this.ingredients = const <Ingredient>[],
     this.probes = const <Probe>[],
     this.probesStandard = true,
     this.probesChecked = false,
     this.saved = false,
-  }) : recipeNumber = recipeNumber ?? const Uuid().v4();
+  })  : recipeNumber = recipeNumber ?? const Uuid().v4(),
+        measurements = measurements ?? [Measurement()];
 
-  Recipe.fromJson(Map<String, dynamic> json)
-      : recipeName = RecipeName.fromJson(json['recipeName']),
-        recipeNumber = json['recipeNumber'],
-        recipeType = json['recipeType'],
-        recipeVolume = RecipeVolume.fromJson(json['recipeVolume']),
-        ingredients = _jsonDecodeIngredients(json['ingredients']),
-        probes = _jsonDecodeProbes(json['probes']),
-        probesChecked = json['probesChecked'] == 'true' ? true : false,
-        probesStandard = json['probesStandard'] == 'true' ? true : false,
-        saved = json['saved'] == 'true' ? true : false;
-
-  Map<String, dynamic> toJson() {
-    final Map<String, dynamic> data = <String, dynamic>{};
-    data['recipeName'] = recipeName.toJson();
-    data['recipeNumber'] = recipeNumber;
-    data['recipeType'] = recipeType;
-    data['recipeVolume'] = recipeVolume.toJson();
-    data['ingredients'] = jsonEncode(ingredients);
-    data['probes'] = jsonEncode(probes);
-    data['probesChecked'] = probesChecked.toString();
-    data['probesStandard'] = probesStandard.toString();
-    data['saved'] = saved.toString();
-    return data;
-  }
-
-  final RecipeName recipeName;
+  final String? recipeName;
   final String recipeNumber;
   final String recipeType;
-  final RecipeVolume recipeVolume;
+  final List<Measurement> measurements;
   final List<Ingredient> ingredients;
   final List<Probe> probes;
-  final bool probesChecked;
-  final bool probesStandard;
+  final bool probesChecked; // Flag to show all recipe probes have been checked
+  final bool
+      probesStandard; // Flag to show all answers to recipe probe match the standard recipe answer
```

Comment:
Similarly, I'd rename this to `allProbeAnswersStandard` or `allProbeAnswersMatchStandardRecipe` and get rid of the comment.